style.css

```diff
@@ -1,140 +1,300 @@
 
-.navbar .hamburger {
-  display: none;
+.nameLogo {
+  margin-left: 1.5rem;
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #5564eb;
+  margin-top: 1rem;
 }
-.navbar .nav-menu {
-  display: inline-flex;
-  list-style: none;
-}
-.navbar .nav-menu li {
+
+.nav-item {
   padding: 10px;
+  width: 62px;
+  height: 20px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #344563;
+}
+
+.bar {
+  display: block;
+  width: 20px;
+  margin: 3px auto;
+  height: 5px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
 }
 
-@media (max-width: 768px) {
-  .navbar .hamburger {
+@media screen and (min-width: 330px) and (max-width: 599px) {
+  .nav-container .hamburger {
     display: block;
+    margin-top: 0;
+    margin-right: 3px;
   }
 
-  .navbar .nav-menu {
+  .nav-container .nav-menu {
     display: none;
   }
 
-  .navbar .hamburger:hover {
+  .nav-container .hamburger:hover {
     cursor: pointer;
   }
 
-  .navbar .hamburger span {
+  .nav-container .hamburger span {
     width: 10px;
     height: 2px;
     background-color: #fff;
     margin: 5px;
     border-radius: 20px;
   }
 
-  .navbar .hamburger span:nth-child(1) {
+  .nav-container .hamburger span:nth-child(1) {
     transform: rotate(45deg);
   }
 
-  .navbar .hamburger span:nth-child(2) {
+  .nav-container .hamburger span:nth-child(2) {
     transform: rotate(-45deg);
   }
 
-  .navbar .hamburger:hover span {
+  .nav-container .hamburger:hover span {
     transform: rotate(0);
   }
 
-  .navbar:hover .nav-menu {
+  .nav-container:hover .nav-menu {
     display: block;
     position: absolute;
     top: 50px;
     left: 0;
     width: 100%;
-    padding: 20px;
+    padding: 40px;
     background-color: #fff;
     z-index: 100;
   }
 
-  .navbar .nav-menu li {
+  .nav-container .nav-menu li {
     list-style-type: none;
     margin: 0;
     padding: 0;
     display: inline;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 20px;
   }
 
-  .navbar .nav-menu li a {
+  .nav-container .nav-menu li a {
     display: block;
     padding: 10px 20px;
     text-decoration: none;
   }
 
-  .navbar .nav-menu li a:hover {
+  .nav-container .nav-menu li a:hover {
     background-color: #ccc;
   }
 
-  .navbar .nav-menu li a:active {
+  .nav-container .nav-menu li a:active {
     background-color: #bbb;
   }
 }
 
-.section2-content{
-  height: 60.4rem;
-   width: 32.7rem; 
-  background-image: url("./assets/header-mobile-view.png");
-  background-repeat: no-repeat;
-  background-size: cover;
-  display: flex;
-  flex-wrap: wrap;
+.section-content {
```

Comment:
- [ ] Kindly as was suggested by the previous reviewers, fix the layout in your design to match with the one provided in Figma

Your design

![image](https://github.com/melkamu372/My-Portfolio/assets/93630700/3b659e06-d4b5-4e79-9852-03a2d1c137fc)

![image](https://github.com/melkamu372/My-Portfolio/assets/93630700/c3d95959-0114-4b7c-b4d2-3cfa37cae0bb)

Figma design

![image](https://github.com/melkamu372/My-Portfolio/assets/93630700/fc38cf85-88f1-47c9-bedd-778a37454478)

Reply:
@BregornOriginal  I solved the lay out about the back ground image I used that is found on the figma.

Reply:
@BregornOriginal  please review it again .

Reply:
Hi @melkamu372 
Have you asked for a review from your dashboard yet? Kindly do that and a reviewer would claim your project 🤗🤗